drivers/SmartThings/zwave-sensor/fingerprints.yml

```diff
@@ -1,9 +1,4 @@
 zwaveManufacturer:
-#   - id: "aeotec/water/6"
```

Comment:
Just confirming, was this intended to be deleted or just uncommented?

Reply:
deleted. it's a duplicate of this already-released fingerprint: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/3388d2ded3b6fcccf4ad5f97fa34832db0688384/drivers/SmartThings/zwave-sensor/fingerprints.yml#L468